### ISSUE TITLE
Incorrect `speed` output after #11755

### ISSUE THREAD

**Issue by romen** [openssl/openssl#11922]
Uhm, I think db71d315479762eefbf2bcda8be3b44b1867133f (from #11755) broke time calculations for `speed`.

```
; util/opensslwrap.sh speed ecdhp224                                                                                                        
Doing 224 bits  ecdh's for 10s: 167278 224-bits ECDH ops in 99999.21s
version: 3.0.0-alpha3-dev
built on: built on: Fri May 22 19:48:41 2020 UTC
options:bn(64,64) rc4(16x,int) des(int) aes(partial) idea(int) blowfish(ptr)
compiler: gcc-9 -fPIC -pthread -m64 -Wa,--noexecstack -Wall -O3 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSL_BUILDING_OPENSSL -DNDEBUG                                                                                   
CPUINFO: OPENSSL_ia32cap=0x7ffaf3ffffebffff:0x29c6fbf
                              op      op/s
 224 bits ecdh (nistp224)   0.5978s      1.7
```

Compare it with the output for alpha2 under the same conditions:

```
; util/opensslwrap.sh speed ecdhp224
Doing 224 bits  ecdh's for 10s: 166748 224-bits ECDH ops in 9.99s
version: 3.0.0-alpha2
built on: built on: Fri May 22 20:49:29 2020 UTC
options:bn(64,64) rc4(16x,int) des(int) aes(partial) idea(int) blowfish(ptr)
compiler: gcc-9 -fPIC -pthread -m64 -Wa,--noexecstack -Wall -O3 -g -DDEBUG_UNUSED -DPEDANTIC -pedantic -Wno-long-long -Wall -Wextra -Wno-unused-parameter -Wno-missing-field-initializers -Wswitch -Wsign-compare -Wshadow -Wformat -Wtype-limits -Wundef -Werror -Wmissing-prototypes -Wstrict-prototypes -Wno-stringop-truncation -DOPENSSL_USE_NODELETE -DL_ENDIAN -DOPENSSL_PIC -DOPENSSL_BUILDING_OPENSSL -DNDEBUG                                                          
CPUINFO: OPENSSL_ia32cap=0x7ffaf3ffffebffff:0x29c6fbf
                              op      op/s
 224 bits ecdh (nistp224)   0.0001s  16691.5
```

db71d315479762eefbf2bcda8be3b44b1867133f seems to have changed the returned value for `Time_F(STOP)`

https://github.com/openssl/openssl/blob/db71d315479762eefbf2bcda8be3b44b1867133f/apps/speed.c#L3258-L3265

https://github.com/openssl/openssl/blob/db71d315479762eefbf2bcda8be3b44b1867133f/apps/speed.c#L174-L180

**Comment by romen** [comment]
@DDvO do you have thoughts on what could have gone wrong?
I spent some time looking at it, but I am not seeing the problem.

**Comment by kroeckx** [comment]
See #11905 and #11903

**Comment by romen** [comment]
Closing as I just noticed this is a duplicate of #11903 and should be fixed with #11905

**Comment by romen** [comment]
> See #11905 and #11903

I even missed @kroeckx saying it!

I'm a shame! ;)
